### ISSUE TITLE
Improvements by using Cloudflare serverless workers

### ISSUE THREAD

**Issue by KennethSweezy** [ScratchUtilities/gobo.cf#70]
I am currently testing a method of making the service quicker by using Cloudflare serverless workers.

If this works could it be called v2?

cc @Andreto

**Comment by Andreto** [comment]
It could be v2 for sure.
How is the stats btw? Do we have users and are links being shortened.

**Comment by KennethSweezy** [comment]
Yes. We do have users, its something like 50 visits a day. But CloudFlare's
analytics aren't very detailed.

**Comment by KennethSweezy** [comment]
@Andreto But with Google Analytics it is something like 6-15 visitors a day. All of them are direct users (not referred to by another website). And it doesn't allow me to view detailed stats for the base domain because it only redirects.

**Comment by retronbv** [comment]
@KennethSweezy is your free domians thing still going on?

Name of Project: retronbv
What is the purpose of your project: personal portfolio, that will eventually have projects on it
Who is/are the owner(s) of the project: me
What is the current web address for the project: profile.retronbv.repl.co
Notes: could i have retronbv.xyz?

(From scratch)

**Comment by sweezyio** [comment]
> @KennethSweezy is your free domians thing still going on?
> 
> Name of Project: retronbv
> What is the purpose of your project: personal portfolio, that will eventually have projects on it
> Who is/are the owner(s) of the project: me
> What is the current web address for the project: profile.retronbv.repl.co
> Notes: could i have retronbv.xyz?
> 
> (From scratch)

Unfortunately, I am no longer providing this service.

**Comment by retronbv** [comment]
Awww ok
